src/VisualStudio/Core/Def/ProjectSystem/CPS/IWorkspaceProjectContext.cs

```diff
@@ -73,6 +74,7 @@ internal interface IWorkspaceProjectContext : IDisposable
 
     void StartBatch();
     IAsyncDisposable CreateBatchScope();
```

Comment:
possibly deprecate.

Reply:
Ditto for `EndBatchAsync`.

Reply:
I added a comment in commit 3 that indicates all three methods that I think can be changed once the managed project system stops consuming them.